src/AElf.Kernel.SmartContract.Parallel/ExecutiveExtensions.cs

```diff
@@ -20,7 +20,7 @@ internal static class ExecutiveExtensions
         {
             var generatedTxn = new Transaction
             {
-                From = Address.Zero,
+                From = Address.FromBytes(new byte[] { }.CalculateHash()),
```

Comment:
make Address.FromBytes(new byte[] { }.CalculateHash()) as a property in a helper method in a test base project.